app/src/main/java/com/codesoom/assignment/optional/Empty.java

```diff
@@ -0,0 +1,6 @@
+package com.codesoom.assignment.optional;
+
+
+public class Empty {
+    public static Empty empty = new Empty();
+}
```

Comment:
null object pattern을 사용하셨군요. `final`로 선언하는 것이 바람직할 것 같습니다.

```suggestion
public class Empty {
    public static final Empty EMPTY = new Empty();
}
```

Reply:
https://github.com/CodeSoom/spring-week2-assignment-1/pull/18/commits/afc1776e924a3b655a87497fed5d6db9f9d4133f 에서 final을 적용하고, 그에따른 이름도 변경해주었어요!